webapp/graphite/render/functions.py

```diff
@@ -503,7 +507,7 @@ def changed(requestContext, seriesList):
         series[i] = 0
   return seriesList
 
-def asPercent(requestContext, seriesList, total=None):
+def asPercent(requestContext, seriesList, seriesList2orNumber=None):
   """
 
   Calculates a percentage of the total of a wildcard series. If `total` is specified,
```

Comment:
Just realized the comments were not updated to reflect the removal of `total`. @cbowman0 would you mind fixing this?